sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/SortAggregateExec.scala

```diff
@@ -87,6 +92,8 @@ case class SortAggregateExec(
           outputIter
         }
       }
+      aggTime += NANOSECONDS.toMillis(System.nanoTime() - beforeAgg)
```

Comment:
is it the right level to trace the agg time? I think the iterator is lazy, no?

Reply:
Also find. it's a little strange...If so HashAggregateExec also incorrect?

Reply:
ping @cloud-fan @holdenk @dongjoon-hyun Could you take a look